wptagent.py

```diff
@@ -1252,6 +1272,7 @@ def main():
     if options.collectversion and platform.system() == "Windows":
         get_browser_versions(browsers)
 
+    options.log_filter = log_filter
```

Comment:
I could be wrong, but I don't know if you can attach arbitrary properties to the options that comes back from argparse.  This is literally just the command-line options.  It is probably better to add another param to the WPTAgent (and WebPageTest) classes and pass the log filter directly.